src/librbd/CMakeLists.txt

```diff
@@ -323,11 +324,11 @@ if(WITH_LTTNG)
 endif()
 
 target_link_libraries(librbd PRIVATE
+  librados
   rbd_internal
   rbd_types
   journal
   libneorados
-  librados
   cls_rbd_client
```

Comment:
Here I would expect the `cls_XYZ_client` static libraries to be moved up between `journal` and `libneorados` (with `librados` after `libneorados`).

Reply:
Thanks a lot for the suggestions. I did try quite a few variations, something similar to this as well but still didn't get it to link so I ended up using librados for rbd_internal but only on Windows. Luckily that's no longer needed with the latest version.